### PR TITLE
API Prototype

### LINKED ISSUE #15: Create a random identifier for each calendar
Each calendar should not only have a unique ID, but a random ID as well. This will help prevent users from navigating to other calendars and modifying the data.

**Acceptance Criteria**
- UUIDs should resemble the form `a1b2c3d4`
- UUIDs shout NOT be the form `0001`

### LINKED ISSUE #21: Create calendar in API
add POST /calendar a calendar with name and given days, returning ID for calendar
add GET /calendar/<id> and return name and days of calendar

The scope of this also jumped to include setup of the repo structure and dev environment, as well as readme to document such, and cover #15

### PR DESCRIPTION
**Pull request by nxb9626**
## Background

Creates a functioning API given the tentative endpoints in the notion outlined [here.](https://www.notion.so/Endpoints-5c4db08ae5e643d08b89b382ae6cd14d#7235eba5e5f94790af23482575758f4a). Also contains setup instructions for the dev environment and some helper scripts for that. 

## Contents
- [x] Implement GET for retrieving a calendar with /calendar/<id\>
- [x] Implement POST for creating a calendar with json body Ex: `{"title": "test", "days": ["monday", "tuesday", "friday"]}`, returning the id

## Testing
1. Follow the instructions to setup the dev environment in the [README.md](https://github.com/noonalu/noonalu-backend/tree/baseline#readme), to the end ensuring all tests pass
2. Pull up postman and go to town on the api (it mostly just parses params right now but there's always edge cases)

### CODE REVIEW

File: setup/destroy.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from setup import *
+import dotenv
+import sys
+
+
+def destroy_database():
```
Comment (cbaudouinjr) [inline setup/destroy.py:6]: Is this code/file required for Mongo to operate? I can't image we'll ever be calling it from production.

Comment (nxb9626) [reply setup/destroy.py:6]: Yeah won't ever be called in production, was just for resetting the dev environment.

File: .env [hunk 2]
```diff
@@ -0,0 +1,2 @@
+MONGO_URI=localhost
+MONGO_PORT=27017
```
Comment (cbaudouinjr) [inline .env]: This file shouldn't be committed. We should include a .env.sample file instead.

**Review by cbaudouinjr — CHANGES_REQUESTED** [review]

File: .env (hunk 2 above)
Comment (nxb9626) [reply .env]: is it ok if it stays in the history? this is the same contents as would go in a .env.sample

Comment (cbaudouinjr) [reply .env]: History is okay. Just need to rename the file.

**Review by cbaudouinjr — APPROVED** [review]
LGTM

**Comment by cbaudouinjr** [comment]
Please be sure to squash and merge.
